drake/doc/developers.rst

```diff
@@ -52,9 +52,12 @@ The following table shows the configurations and platforms that Drake
 officially supports. Supported configurations are tested in continuous
 integration. All other configurations are provided on a best-effort basis.
 
-On Ubuntu and OS X, only the "Unix Makefiles" CMake generator is supported.
-On Windows, only the "Visual Studio 14 2015" and "Visual Studio 14 2015 Win64"
-CMake generators are supported.
+On Ubuntu and OS X, the "Unix Makefiles" and "Ninja" CMake generators are
+supported. On Windows, the "Visual Studio 14 2015",
+"Visual Studio 14 2015 Win64", and "Ninja" CMake generators are supported.
+
+The supported version of MATLAB is R2015b. The supported version of the
+Java Development Kit is 7 on Windows and Ubuntu, and 8 on OS X.
```

Comment:
Personally, I would just say JDK 7 for all platforms. I can tweak the OS X configuration on CI easily enough. Much of me would like to say that the minimum version is also 7 for all platforms, which realistically would mean R2013b would be minimum MATLAB version.